### PR TITLE
Fix BZ 64532 - update to bnd 5.1.1

### PR DESCRIPTION
**Pull request by rotty3000**
Signed-off-by: Raymond Augé <rotty3000@apache.org>

### CODE REVIEW

File: build.properties.default [hunk 1]
```diff
@@ -286,14 +286,14 @@ saaj-api.home=${base.path}/saaj-api-${saaj-api.version}
 saaj-api.jar=${saaj-api.home}/saaj-api-${saaj-api.version}.jar
 saaj-api.loc=${base-maven.loc}/javax/xml/soap/saaj-api/${saaj-api.version}/saaj-api-${saaj-api.version}.jar
 
-# ----- bnd & bndlib, version 4.0.0 or later  -----
+# ----- bnd, version 5.1.1 or later  -----
 # ----- provides OSGI metadata for JARs       -----
-bnd.version=5.1.0
+bnd.version=5.1.1
 
 # checksums for biz.aQute.bnd-5.1.0.jar
```
Comment (markt-asf) [inline build.properties.default:293]: You need to update the version in the comment for consistency.

File: build.properties.default [hunk 2]
```diff
@@ -286,14 +286,14 @@ saaj-api.home=${base.path}/saaj-api-${saaj-api.version}
 saaj-api.jar=${saaj-api.home}/saaj-api-${saaj-api.version}.jar
 saaj-api.loc=${base-maven.loc}/javax/xml/soap/saaj-api/${saaj-api.version}/saaj-api-${saaj-api.version}.jar
 
-# ----- bnd & bndlib, version 4.0.0 or later  -----
+# ----- bnd, version 5.1.1 or later  -----
```
Comment (markt-asf) [inline build.properties.default:289]: Is this really an increase in the minimum version? Or is 5.1.0 (with the fix for the Java 15 bug) sufficient?

Comment (rotty3000) [reply build.properties.default:289]: There is that yes, but also in 5.1.1 specifically there is a bug fix for correctly generating the Manifest Name section which I'd like to use for the Spec jars.

Comment (markt-asf) [reply build.properties.default:289]: Fair enough. Just checking.

File: build.properties.default (hunk 1 above)
Comment (rotty3000) [reply build.properties.default]: Done!

**Comment by rotty3000** [comment]
@markt-asf did I miss any other changes you wanted to see?

**Pull request merged**